game-backend/src/MainController.ts

```diff
@@ -41,8 +42,10 @@ export default class MainController {
 
     // generate a unit
     const unitId = this.hyperIdInstance();
-    const unit: GameObject = {
+    const unit: Unit = {
       id: unitId,
+      type: GAME_OBJECT_TYPES.UNIT,
+      health: 100,
```

Comment:
Should we rename energy to health or the other way around?

Reply:
I would prefer health, energy might later be something else

Reply:
Okay, please update the stories accordingly.